apps/builder/app/shared/copy-paste/plugin-instance.ts

```diff
@@ -93,11 +97,19 @@ const getTreeData = (targetInstanceSelector: InstanceSelector) => {
     treeStyleSourceIds
   );
 
-  const treeStyles = getMapValuesBy(stylesStore.get(), (styleDecl) =>
-    treeStyleSourceIds.has(styleDecl.styleSourceId)
+  const treeBreapointIds = new Set<Breakpoint["id"]>();
+  const treeStyles = getMapValuesBy(stylesStore.get(), (styleDecl) => {
+    treeBreapointIds.add(styleDecl.breakpointId);
+    return treeStyleSourceIds.has(styleDecl.styleSourceId);
+  });
```

Comment:
Seem like we're putting into `treeStyles` only styles that satisfy `treeStyleSourceIds.has()`, but we put into `treeBreapointIds` **all** breakpoints. Is this a bug?

Reply:
It's filter function which iterates whole map. Just didn't want to add the second iteration.

Reply:
Yeah, but does this mean we're copying and merging all breakpoints?

Reply:
we should def. only copy breakpoints that are used on the copied instances

Reply:
You are right. I get what you mean now. Will add test